<launch>
	
	<node name="video_feed" pkg="tshroom" type="video_feed" output="screen">
		<param name="camera_port" value="0" />
		<param name="camera_width" value="640" />
		<param name="camera_height" value="480" />
		
		<param name="v4l2_configuration" value="0" />
	</node>
	
	<node name="image_processing" pkg="tshroom" type="image_processing" output="screen">
		<param name="input_path" value="/home/aldwinakbar/Dropbox/Projects/campus/team/tShroom/initial_development/tShroom_480p.mp4" />
		<!-- <param name="input_path" value="" /> -->
		<param name="display_video" value="0" />
	</node>
	
	<node name="arduino" pkg="rosserial_server" type="serial_node" args="_port:=/dev/ttyUSB0" />
	
</launch>
